editor/questionbankcontroller.js

```diff
@@ -6,6 +6,8 @@
 'use strict';
 angular.module('createquestionapp', [])
   .controller('QuestionFormController', ['$scope', 'pluginInstance', function ($scope, pluginInstance) {
+    var savedFilters = null;
+    var savedQuestions = null;
```

Comment:
Why are you using variable not the scope level variable in angular

Reply:
Because $scope is getting overridden the closure is failing. If we use a local variable, it is always preserved.

This is the solution for the unshift error.